data/deb/postinst.sh

```diff
@@ -70,7 +70,7 @@ install_mroonga() {
     need_manual_update=yes
   fi
 
-  install_plugin_sql=/usr/share/mroonga/install_plugin.sql
```

Comment:
Can we remove `data/install_plugin.sql.in`?

Reply:
Why do we can remove data/install_plugin.sql.in?
install.sql call install_plugin.sql. Therefore, I think we need data/install_plugin.sql.in.

Reply:
If this was the last place that refers `install_plugin.sql` except `install.sql`, we don't need to split `install_plugin.sql` from `install.sql`.
We can merge `install_plugin.sql` into `install.sql`.